### PR TITLE
Add 3.0-upg version of MongoDB.

### PR DESCRIPTION
**Pull request by omron93**
This PR adds a new versions of container: centos/mongodb-30-upg-centos7 (name came from https://github.com/sclorg/mongodb-container/pull/127#issuecomment-218879835)

Added notice about that this image is supported only for upgrade from centos/mongodb-26-centos7 to centos/mongodb-32-centos7 (added in README.md and printed after container start).

Only small changes from 2.6 version - [26-to-30-upg-diff.txt](https://github.com/sclorg/mongodb-container/files/434200/26-to-30-upg-diff.txt)

@hhorak @bparees Please take a look and merge.

### CODE REVIEW

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
I guess the new version should be added to the `Makefile` and `README.md`.

**Comment by omron93** [comment]
@hhorak Added, thanks.

**Comment by hhorak** [comment]
[test]

**Comment by hhorak** [comment]
Now it looks fine to me, so let's wait for test to finish.

File: 3.0-upg/root/usr/bin/run-mongod [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    pkill -2 mongod
+  fi
+  wait_for_mongo_down
+  exit 0
+}
+
+echo "
+
+
+Notice: This image is supported only for upgrade from centos/mongodb-26-centos7 to centos/mongodb-32-centos7.
```
Comment (php-coder) [inline 3.0-upg/root/usr/bin/run-mongod:59]: If I run `rhscl/mongodb-30-upg-rhel7` image I also will see this message?

Comment (omron93) [reply 3.0-upg/root/usr/bin/run-mongod:59]: @php-coder Probably yes. Any idea how to change this?

Comment (php-coder) [reply 3.0-upg/root/usr/bin/run-mongod:59]: @omron93 `Notice: This image is supported only for upgrade from MongoDB 2.6 to 3.2` ?

**Comment by bparees** [comment]
@hhorak @omron93 
are you building+pushing the centos image to dockerhub or do you need me to update our infrastructure to do so?

**Comment by omron93** [comment]
@bparees I don't know, Honza could have an answer. He has PTO, so probably on Monday.

**Pull request merged**